analyses/interaction-plots/scripts/cooccur_functions.R

```diff
@@ -79,12 +79,32 @@ filter_mutations <- function(maf_df,
 #'   samples present in the data frame.
 #'
 #' @return A data frame summarizing the co-occurence of pairs of genes in the
-#'   gene list with columns `gene1`; `gene2`; counts of each mutations in
-#'   each category of sharing (`mut11`: both mutated; `mut10`: mutated in
-#'   the first but not second gene, etc.); `odds_ratio` for co-occurence,
-#'   `cooccur_sign`: 1 if co-occurence greater than by chance, -1 if less
-#'   frequent than expected; `p` the fisher's exact test p value; and a
-#'   `cooccur_score` calucated as `cooccur_sign * -log10(p)`.
+#'   gene list with columns:
+#'  `gene1`; `gene2` (gene names);
+#'  `mut11`; `mut10`; `mut01` (counts of mutations in each category of sharing:
+#'     `mut11`: both mutated; `mut10`: mutated in the first but not second gene, etc.);
+#'  `odds_ratio` (odds ratio for for co-occurence);
+#'  `cooccur_sign` (1 if co-occurence greater than by chance, -1 if less frequent than expected)
+#'  `p` (the fisher's exact test p value);
+#'  `q` (Benjamini-Hochberg adjusted p value);
+#'  `cooccur_score` (calculated as `cooccur_sign * -log10(p)`);
+#'  `n_mutated_gene1`; `n_mutated_gene2` (count of samples with gene1 or gene2 mutations); 
+#'  `perc_mutated_gene1`; `perc_mutated_gene2` (percent of samples with gene 1 or gene 2 mutations);
+#'  `perc_cooccur`;
+#'  `perc_mutexl`;#' @return A data frame summarizing the co-occurence of pairs of genes in the
+#'   gene list with columns:
+#'  `gene1`; `gene2` (gene names);
+#'  `mut11`; `mut10`; `mut01` (counts of mutations in each category of sharing:
+#'     `mut11`: both mutated; `mut10`: mutated in the first but not second gene, etc.);
+#'  `odds_ratio` (odds ratio for for co-occurence);
+#'  `cooccur_sign` (1 if co-occurence greater than by chance, -1 if less frequent than expected)
+#'  `p` (the fisher's exact test p value);
+#'  `q` (Benjamini-Hochberg adjusted p value);
+#'  `cooccur_score` (calculated as `cooccur_sign * -log10(p)`);
+#'  `n_mutated_gene1`; `n_mutated_gene2` (count of samples with gene1 or gene2 mutations); 
+#'  `perc_mutated_gene1`; `perc_mutated_gene2` (percent of samples with gene 1 or gene 2 mutations);
+#'  `perc_cooccur`; (Percent of all surveyed samples with co-occurring gene1/2 mutations when either gene1 or gene2 are mutated);
+#'  `perc_mutexcl`; (Percent of all surveyed samples with gene1/2 mutations being mutually exclusive);
```

Comment:
I was confused by the "all surveyed samples" here, so I'm trying a slightly different wording here.
```suggestion
#'  `perc_cooccur`; (Percent of samples with mutations in gene 1 and/or gene 2 with co-occurring gene1/2 mutations);
#'  `perc_mutexcl`; (Percent of samples with mutations in gene 1 and/or gene 2 where gene1/2 mutations are mutually exclusive);
```